docs/content/development/extensions-contrib/tdigestsketch-quantiles.md

```diff
@@ -133,7 +104,7 @@ This returns an array of quantiles corresponding to a given array of fractions.
 |--------|-----------|---------|
 |type|This String should always be "quantilesFromTDigestSketch"|yes|
 |name|A String for the output (result) name of the calculation.|yes|
-|fieldName|A String for the name of the input field containing raw numeric values.|yes|
+|field|A field reference pointing to the field aggregated/combined T-Digest sketch.|yes|
```

Comment:
Suggest rewriting to:

"The input field reference, which must be a PostAggregator that outputs T-Digest sketches. This can be a `fieldAccess` PostAggregator, which simply returns the value of the referenced input column."